### PR TITLE
Test deployment 

### PR DESCRIPTION
**Pull request by Ciuffi**
Right now this will deploy to a github url, probably best to test it before we switch it to the official domain.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# LHD2020
+# NwPlus2020
```
Comment (luceliu) [inline README.md]:
```suggestion
# nwPlus2020
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-> The LHD website
+> The NwPlus website
```
Comment (luceliu) [inline README.md]:
```suggestion
> The nwPlus website
```

File: functions/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin'
+admin.initializeApp()
+const Mailchimp = require('mailchimp-api-v3');
+const API_KEY: string = functions.config().mailchimp.key
+const mailchimp = new Mailchimp(API_KEY)
+// // Start writing Firebase Functions
+// // https://firebase.google.com/docs/functions/typescript
+//
+export const SubscribeToMailingList = functions.https.onRequest(async (request, response) => {
```
Comment (luceliu) [inline functions/src/index.ts]:
```suggestion
export const subscribeToMailingList = functions.https.onRequest(async (request, response) => {
```

File: package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "LHD2020",
+  "name": "nwPlus",
   "version": "1.0.0",
-  "description": "The LHD website",
+  "description": "The NwPlus Website website",
```
Comment (luceliu) [inline package.json]:
```suggestion
  "description": "The nwPlus website",
```

File: package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
   "author": "Nwplus Dev Team",
```
Comment (luceliu) [inline package.json]:
```suggestion
  "author": "nwPlus Dev Team",
```

**Comment by luceliu** [comment]
Sorry for all the capitalization commits, just being anal

**Comment by Ciuffi** [comment]
@luceliu all good I like it better this way

**Pull request merged**